src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.LogicApps.tests.ps1

```diff
@@ -678,4 +710,467 @@ InModuleScope Arcus.Scripting.LogicApps {
+                    return @{
+                        SubscriptionId = "123456"
+                        AccessToken = "accessToken"
+                    }
+                }
+
+                # Act
+                Disable-AzLogicAppsFromConfig -DeployFileName "$PSScriptRoot\Files\LogicApps\Standard\deploy-orderControl-noWaitingOrRunningRunsWithNoneStopType.json" -ResourceGroupName $resourceGroup
+
+                # Assert
+                Assert-MockCalled Get-AzCachedAccessToken -Scope It -Times 1
+                Assert-MockCalled Invoke-WebRequest -Scope It -Exactly 6
+                Assert-MockCalled Disable-AzLogicApp -Scope It -Exactly 0
+            }
+            It "Disables all logic apps when checkType = NoWaitingOrRunningRuns with found waiting and no running runs for stopType = Immediate" {
+                # Arrange
+                $script:i = 0
+                $resourceGroup = "my-resource-group"
+                $logicAppNames = @("snd-async", "ord-sthp-harvest-order-doublechecker", "rcv-sthp-harvest-order-af-ftp", "rcv-sthp-harvest-order-af-sft", "rcv-sthp-harvest-order-af-file")
+                
+                Mock Invoke-WebRequest -MockWith {
+                    $script:i++
+                    if ($script:i -gt 2) {
+                        Write-Host "Returning empty (no running, no waiting) runs"
+                        return [pscustomobject] @{
+                            Content = '{"value":[]}'
+                        }
+                    } else {
+                        Write-Host "Returning 1 waiting run"
+                        return [pscustomobject] @{
+                            Content = '{"value":[{"properties":{"status":"Waiting"}}]}'
+                        }
+                    }
+                }
+                Mock Disable-AzLogicApp {
+                    $ResourceGroupName | Should -Be $resourceGroup
+                    $LogicAppName | Should -BeIn $logicAppNames
+                }
+                Mock Get-AzCachedAccessToken -MockWith {
+                    $Global:subscriptionId = '123456'
+                    return @{
+                        SubscriptionId = "123456"
+                        AccessToken = "accessToken"
+                    }
+                }
+
+                # Act
+                Disable-AzLogicAppsFromConfig -DeployFileName "$PSScriptRoot\Files\LogicApps\Standard\deploy-orderControl-noWaitingOrRunningRunsWithImmediate.json" -ResourceGroupName $resourceGroup
+
+                # Assert
+                Assert-MockCalled Get-AzCachedAccessToken -Scope It -Times 1
+                Assert-MockCalled Invoke-WebRequest -Scope It -Exactly 12
+                Assert-MockCalled Disable-AzLogicApp -Scope It -Exactly 5 -ParameterFilter { $resourceGroupName -eq $resourceGroup }
+            }
+            It "Disables all logic apps when checkType = NoWaitingOrRunningRuns with found waiting and running runs for stopType = Immediate" {
```

Comment:
Think these tests are all the same like we already had, right? If not, could you point me the new ones? Thx.

Reply:
For Standard I implemented the same tests as Consumption.
Consumption tests are here: https://github.com/pim-simons/arcus.scripting/blob/feature/logic-app-standard/src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.LogicApps.tests.ps1#L5
Standard tests are here: https://github.com/pim-simons/arcus.scripting/blob/feature/logic-app-standard/src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.LogicApps.tests.ps1#L713